connector-packager/connector_packager/xsd_validator.py

```diff
@@ -285,14 +285,17 @@ def validate_file_specific_rules_tdr(file_to_test: ConnectorFile, path_to_file:
 
         if len(attributes) > 0 and properties.connection_fields:
             for field in properties.connection_fields:
+                if field == 'instanceurl':
+                    continue
                 if field not in attributes:
                     xml_violations_buffer.append("Attribute '" + field + "' in connection-fields but not in required-attributes list.")
                     return False
 
         if len(attributes) > 0 and properties.connection_metadata_database and not properties.uses_tcd:
             if 'dbname' not in attributes:
                 xml_violations_buffer.append("connection-metadata 'database' enabled but 'dbname' not in required-attributes list.")
-                return False
+        
```

Comment:
Since we're no longer rejecting the connector if dbname is in the wrong place, move this logic to [warn_file_spicific_rules_tdr](https://github.com/tableau/connector-plugin-sdk/blob/master/connector-packager/connector_packager/xsd_validator.py#L343) and use logger.warning rather than adding to xml_violations_buffer to be consistent with other warnings

Reply:
Updated this to the warn_file_spicific_rules_tdr and added a warning to the top.